pulsar-client-cpp/lib/MultiTopicsConsumerImpl.cc

```diff
@@ -54,6 +55,39 @@ MultiTopicsConsumerImpl::MultiTopicsConsumerImpl(ClientImplPtr client, const std
     }
 }
 
+MultiTopicsConsumerImpl::MultiTopicsConsumerImpl(ClientImplPtr client, const std::string singleTopic,
```

Comment:
Creating another constructor just for the MultiTopicsConsumer seems to create a lot of duplicate codes. I think we can create a static method to create a partitioned topic consumer based on the multi topics consumer.

Reply:
Thanks, fixed.